### PR TITLE
SAK-29488 Populated access.scss

### PR DESCRIPTION
**Pull request by adrianfish**

### CODE REVIEW

**Comment by adrianfish** [comment]
@SedueRey There you go ...

File: reference/library/src/morpheus-master/sass/access.scss [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+            left:2px;
+            margin-bottom:2em;
+
+            a {
+                border:1px solid #ccc;
+                padding:3px;
+                display: inline-block;
+            }
+
+            a:hover {
```
Comment (SedueRey) [inline reference/library/src/morpheus-master/sass/access.scss]: Just for the record, you can do this to this way:

     a {
                border:1px solid #ccc;
                padding:3px;
               display: inline-block;
               &:hover{
                          padding-left:21px;
                          background: url(../../image/silk/folder.png) center left no-repeat !important
               }
    } 

Don't repeat this, of course, it's just to say.

**Comment by SedueRey** [comment]
Looks good to me :)

**Comment by adrianfish** [comment]
Thanks for the review @SedueRey 

I've amended as per your advice :)

**Pull request merged**